tests/pages/formLogin.py

```diff
@@ -0,0 +1,26 @@
+from selenium.webdriver.common.by import By
+
+from .common import CommonOps
+
```

Comment:
wrong naming convention - file name should be snake case

Reply:
ok, it is done